src/drivers/gnss/septentrio/septentrio.cpp

```diff
@@ -1199,20 +1199,120 @@ int SeptentrioDriver::process_message()
 
 			if (_sbf_decoder.parse(&receiver_status) == PX4_OK) {
 				_message_gps_state.rtcm_msg_used = receiver_status.rx_state_diff_corr_in ? sensor_gps_s::RTCM_MSG_USED_USED : sensor_gps_s::RTCM_MSG_USED_NOT_USED;
+
+				_message_gps_state.system_error = sensor_gps_s::SYSTEM_ERROR_OK;
+
+				if (receiver_status.rx_error_cpu_overload)
```

Comment:
This is minor, but can you make sure that even one line if statements have braces to prevent any accidents?

```suggestion
				if (receiver_status.rx_error_cpu_overload) {
```

Reply:
I've added all the missing braces